java/core/src/main/java/com/whylogs/core/PreProcessedColumn.java

```diff
@@ -0,0 +1,68 @@
+package com.whylogs.core;
+
+import lombok.*;
+
+import java.util.Collection;
+import java.util.List;
+
+@Getter @Setter @EqualsAndHashCode
+public class PreProcessedColumn {
```

Comment:
I think it's `PreprocessedColumn` in Python

Reply:
it's preprocessed_column so yes going to camel cause I thinnk PreProcessed is easier to read and still the same name.

Reply:
Richard is correct on the class name, in python it is PreprocessedColumn: https://github.com/whylabs/whylogs/blob/mainline/python/whylogs/core/preprocessing.py#L69